### ISSUE TITLE
Students can skip individual review questions by spamming the arrow key while completing the last few chapter question

### ISSUE THREAD

**Issue by Brooke-Fail** [openstax/tutor#634]
### Description
Students can use the arrow keys to move through a reading. If they answer the questions quickly, and use the arrow keys to move forward faster than the next pages are loading, they can skip 3-4 questions.

**To Reproduce**
Steps to reproduce the behavior are
1. Log into tutor as a student
2. Go to a course that has short reading (1 chapter)
3. Quickly spam the arrow keys to move forward and answer any questions as quickly as possible
4. Notice that the student can skip 3-4 questions at the end

**Expected behavior**
Reading questions can not be skipped

**Screenshots**
Video:
https://drive.google.com/file/d/1cQ2g9EZs1m4LJ5XyWf7hdAXkkokOxL75/view?usp=sharing

**Additional context**
- Issue can also be reproduced from the last section of the reading. 

### Acceptance Tests
Additional acceptance tests that are required by this bug fix.

**title**: template  
**categories**: Interactive Component, Navigation, TOC, Math, Book Content, Other
> GIVEN something  
> AND something else  
> WHEN something  
> AND something else  
> THEN something  
> AND something else  

### Checklist for Done
2-CODE
  - [ ] A pull request is opened and linked to this issue. The automated pull request checks pass. 
  - [ ] The change has been approved by other developers.
  - [ ] The pull request is merged into master, and the change branch is deleted.
  - [ ] If there are new acceptance tests, they are finalized in "Given / When / Then" format.
  - [ ] Regression test categories are identified on the issue. Categories should be referenced if any changes might affect them, even if the intended functionality is unchanged.
- [ ] Add milestone tag
  
4A-UX REVIEW
- [ ] If UI changes, UX reviews and verifies. 

5A-FUNCT VER
  - [ ] All the acceptance tests for this issue have passed.
 
5C-REGRESSION
  - [ ] The test plan in testrail passes

**Comment by nathanstitt** [comment]
I **think** this has been fixed, at least I can't seem to replicate it

**Comment by Brooke-Fail** [comment]
Still occurring in QA.

**Comment by Brooke-Fail** [comment]
I'm still able to reproduce this in QA. I see that the arrows pop up later when it has loaded, but I can still skip by spamming (just takes longer).

**Comment by nathanstitt** [comment]
@carlycurran what I think is going on here is that reading steps can be advanced through very quickly.

Would it be ok to require that a step be displayed for say 2 seconds before it can be skipped?  that would prevent this

**Comment by carlycurran** [comment]
Ha! Yes, I would love that (as would research I'm guessing). What will happen if they try to advance at 1 sec? nothing?

**Comment by nathanstitt** [comment]
yeah, nothing.  we'd deactivate all the controls until the timeout had been reached

**Comment by carlycurran** [comment]
yes! we can only delay if this is the first time a student has seen the step.

**Comment by Brooke-Fail** [comment]
I am still able to skip 1 or 2 individual review questions. If you look at here the grey arrows are, you can see me skip a question. 

https://drive.google.com/file/d/1jfdbP_2gB__QwawWVcCIWZlnyZe6jZhw/view?usp=sharing
